_data/students.yml

```diff
@@ -444,3 +444,11 @@
   image: rahul.jpg
   twitter: 448bb8
   bio: Programmer
+
+  - name: Naveen Balachandran
```

Comment:
There are build errors because of the extra space before 'name'. 
Remove it and this is all set to be merged! :)